src/ripple/protocol/impl/STTx.cpp

```diff
@@ -354,11 +356,11 @@ STTx::checkMultiSign () const
             validSig = false;
         }
         if (!validSig)
-            return false;
+            return {false, "Invalid signature."};
```

Comment:
Would it be worth including the hex value of `accountID` here? I'd hate to go through the hassle of wrangling a dozen signers only to get this message and not know who to yell at.